core/vnl/vnl_matrix_fixed.h

```diff
@@ -111,6 +111,19 @@ class VNL_TEMPLATE_EXPORT vnl_matrix_fixed
 
   //: Construct an empty num_rows*num_cols matrix
   vnl_matrix_fixed() = default;
+  //: Construct an m*n Matrix and copy rhs into it.
+  //  Abort if rhs is not the same size.
+  vnl_matrix_fixed(const vnl_matrix_fixed& rhs) = default;
+  vnl_matrix_fixed(vnl_matrix_fixed&& other) = default;
+  //: Copy another vnl_matrix_fixed<T,m,n> into this.
+  vnl_matrix_fixed& operator=(const vnl_matrix_fixed& rhs) = default;
+  vnl_matrix_fixed& operator=(vnl_matrix_fixed&& rhs) = default;
+  //  Destruct the m*n matrix.
+  // An explicit destructor seems to be necessary, at least for gcc 3.0.0,
```

Comment:
Comment no longer relevant.

Reply:
Done, + many other gcc 3.0 compiler references in comments removed.